deploy/scripts/src/cri-kiwi/test.ts

```diff
@@ -78,19 +78,47 @@ export function setup (): void {
 }
 
 const env = {
-  envURL: __ENV.IDENTITY_KIWI_URL
+  envURL: __ENV.IDENTITY_KIWI_URL,
+  envIPVStub: __ENV.IDENTITY_KIWI_STUB_URL,
+  envVerifyRequest: __ENV.IDENTITY_KIWI_VERIFY_REQUEST,
+  envTarget: __ENV.IDENTITY_KIWI_TARGET
 }
 
 const transactionDuration = new Trend('duration')
 
 export function CIC (): void {
   let res: Response
   let csrfToken: string
+  let requestValue: string
+  let clientId: string
+  const obj = {
+    target: env.envTarget
+  }
+
+  const payload = JSON.stringify(obj)
 
-  group('B01_CIC_01_LaunchLandingPage GET', function () {
+  group('B01_CIC_01_IPVStubCall POST', function () {
     const startTime = Date.now()
-    res = http.get(env.envURL, {
-      tags: { name: 'B01_CIC_01_LaunchLandingPage' }
+    res = http.post(env.envIPVStub + '/start', payload,
+      {
+        tags: { name: 'B01_CIC_01_IPVStubCall' }
+      })
+    const endTime = Date.now()
+    check(res, {
+      'is status 201': (r) => r.status === 201,
+      'verify page content': (r) =>
+        (r.body as string).includes(env.envVerifyRequest)
+    })
+      ? transactionDuration.add(endTime - startTime)
+      : fail('Response Validation Failed')
+    requestValue = getRequestCode(res)
+    clientId = getClientID(res)
+  })
+
+  group('B01_CIC_02_Authorize GET', function () {
+    const startTime = Date.now()
+    res = http.get(env.envURL + '/oauth2/authorize?request=' + requestValue + '&response_type=code&client_id=' + clientId, {
```

Comment:
```suggestion
    res = http.get(env.envURL + '/oauth2/authorize',
    {
      request: requestValue,
      response_type: code,
      client_id: clientId
    },
    {
```
This would be more readable in `object` format